client/.env

```diff
@@ -1 +1,3 @@
-VITE_API_URL=http://localhost:8000
\ No newline at end of file
+VITE_API_URL=http://localhost:8000/api
```

Comment:
updated local backend to include the /api for getting form metadata and events, but I still need to validate this is working as expected. The function backend seems to default to adding `/api` to the endpoint so that's why this change was introduced. We can look into if we are able to change the default for functions if we need to